src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static class HostBuilderExtensions
 		/// <summary>
 		/// Registering Dependency Injection classes that will provide Service Fabric specific information for logging
 		/// </summary>
-		public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this IServiceCollection collection)
+		public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this IServiceCollection collection, HostBuilderContext? hostBuilderContext)
```

Comment:
I don't like this approach.
It is messy.
Given that it is a breaking change for Initialization Logger, I'd be in flavor of making it a fully breaking change - updating the package version would be disabling the old logger. Since both approaches would require a code change and deployment to release, there is no release operational difference.
The main difference is that downstream consumers would need to be aware of the breaking change in the package update.

Reply:
Alternatively, define extra extension methods where one adds the new dependencies and the other adds the old dependencies, then move the configuration handling back to the service and out of the dependency registration extensions.

Reply:
I would be in faivour of doing breaking change and making `AddOmexServiceFabricDependencies` an extension on `HostBuilderContext`, since pattern of optional parameter super unusial and easy to miss for user. We can still keep old extension method if absolutly needed but mark it as obsolete with an explanation of why change needed.